qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -162,6 +162,124 @@ def test_mkdir_path_with_path_option(self):
         o = self.mount_a.stat('d5/d6/d7')
         log.info("mount_a output:\n{}".format(o))
 
+class TestRMdir(TestCephFSShell):
+    def test_rmdir(self):
+        """
+        Test that rmdir deletes directory
+        """
+        o = self.get_cephfs_shell_cmd_output("mkdir test_dir")
+        log.info("cephfs-shell output:\n{}".format(o))
+
+        o = self.get_cephfs_shell_cmd_output("rmdir test_dir")
+        log.info("cephfs-shell output:\n{}".format(o))
+
+        try:
+            o = self.mount_a.stat('test_dir')
+            log.info("mount_a output:\n{}".format(o))
+            raise Exception("Something went wrong!! test_dir is not deleted")
+        except CommandFailedError:
```

Comment:
Looking for `CommandFailedError` is not a nice way. Often, other kind of failures leading to `CommandFailedError` gets misinterpreted as valid failure.